tests/unit/model/test_model.py

```diff
@@ -293,6 +296,40 @@ def test_feature_columns(object_under_test):
     assert object_under_test.feature_columns == expected
 
 
+def test_transform_columns():
```

Comment:
This needs construct a `Model` object, ideally via the `from_api_repr` method as above.